scripts/.air_s3.toml

```diff
@@ -0,0 +1,23 @@
+root = "."
+tmp_dir = ".air"
+
+[build]
+  full_bin = "./.air/bytebase --port 8080 --frontend-port 3000 --data . --backup-bucket s3:us-east-1//bytebase-lyl-dev --credentials-file ~/.aws/credentials"
```

Comment:
just nit, if naming dev,  dev-xxx is preferable than xxx-dev, since prefix format is more intuitive.

And for this one, since it's called air_s3.toml instead of air_s3_lyl.toml, then we may call this bytebase-dev-backup